src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -247,6 +247,33 @@ public void Initialize()
                 }
             };
 
+            try
+            {
+                if (MemoryInformation.IsSwappingOnHddInsteadOfSsd())
+                {
+                    var alert = AlertRaised.Create("Swap Storage Type Warning",
+                        "OS swapping on at least one HDD drive while there is at least one SSD drive on this system. " +
+                        "This can cause a slowdown, consider moving swap-partition/pagefile to SSD",
+                        AlertType.SwappingHddInsteadOfSsd,
+                        NotificationSeverity.Warning,
+                        "Swap Storage Type Warning");
```

Comment:
AlertType.SwappingHddInsteadOfSsd already gives you uniqueness. The last parameter is used when you want to distinguish operations which can be multiple, like: in SQL replication we set type to AlertType.SqlConnection error and then as last parameter we pass sql replication setup name. As result unique key is created using 2 fields.